example/lib/device_details/device_details_bloc.dart

```diff
@@ -232,7 +233,8 @@ class DeviceDetailsBloc {
   }
 
   void dispose() async {
-    _deviceController.value?.abandon();
+    await connectionSubscription.cancel();
```

Comment:
I can't find any place where this field would be set in this file?

Reply:
You're right! One more thing to remove. :D I have added `cancel()` because VSC shows a warning when streams are not cancelled.